ansible/group_vars/network/devnet.yml

```diff
@@ -1,3 +1,4 @@
+dev_id: devnet
```

Comment:
We shouldn't default this here. The idea is to force users to have it in their inventory so, it can generate unique chain ids and avoid S3 conflicts. E.g.: if you see my config file `dev_id: emanuel`. You should have this inside your inventory.